packages/vega-typings/types/runtime/index.d.ts

```diff
@@ -52,8 +60,8 @@ export class View {
   loader(): Loader;
   loader(loader: Loader): this;
 
-  locale(): Locale;
-  locale(locale: Locale): this;
+  locale(): LocaleFormatters;
+  locale(time: TimeLocale, number: NumberLocale): this;
```

Comment:
This change is incorrect. The Dataflow (View’s Patent class) takes a combined locale formatters object as input: https://github.com/vega/vega/blob/master/packages/vega-dataflow/src/dataflow/Dataflow.js#L85

Reply:
Ahh gotcha. I got confused by the [Locale API](https://vega.github.io/vega/docs/api/locale/) docs. Reverted.

Reply:
Let's add the correct typings for `vega.locale` as well then.